codechecker_common/util.py

```diff
@@ -93,111 +89,6 @@ def get_binary_in_path(basename_list, versioning_pattern, env):
         return files[0]
 
 
-def kill_process_tree(parent_pid, recursive=False):
```

Comment:
Why duplicate ```kill_process_tree```, as it's used in both modules?

Reply:
It is duplicated so there is no dependency between the analyzer and the web part.